etc/.dockerfilelintrc

```diff
@@ -2,3 +2,8 @@
 rules:
   # Workaround until replicatedhq/dockerfilelint#169 is resolved
   missing_tag: off
+  # Two apt-get exemptions below are to prioritize component updates/maintenance over hermetically sealed, reproducable builds
+  # Permit apt-get update to be used generally
+  apt-get-update_require_install: off
```

Comment:
Apt update just updates the local package index for later installs. I don't know of any situation where we would want to do an update but not as install. Also, since we delete the package index at the end of a RUN, a new update would be required for later installs, meaning this rule is always valuable to leave to the default on, as far as I can tell 

Was there a use case you were thinking of that I'm not?

Reply:
This is resolved - we added a package with `apt-get install`